File: kindergarden-Api/src/main/java/org/wecancodeit/kindergarden/models/Parent.java
package org.wecancodeit.kindergarden.models;

import java.util.Collection;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.OneToMany;

@Entity
public class Parent {


@Id
@GeneratedValue
private Long id;
private	String firstName;
private String lastName;
private String phoneNumber;
private String email;

@OneToMany(mappedBy ="parent")
private Collection<Child> childrens;

public Parent() {}


public Parent(String firstName, String lastName, String phoneNumber, String email) {
	this.firstName = firstName;
	this.lastName = lastName;
	this.phoneNumber = phoneNumber;
	this.email = email;
	
}

public Long getId() {
	return id;
}
public String getFirstName() {
	return firstName;
}

public String getLastName() {
	return lastName;
}
public String getPhoneNumber() {
	return phoneNumber;
}
public String getEmail() {
	return email;
}

public Collection<Child> getChild() {
	return childrens;
}
@Override
public String toString() {
	return "Parent [id=" + id + ", firstName=" + firstName + ", lastName=" + lastName + ", phoneNumber=" + phoneNumber
			+ ", email=" + email + ", children=" + childrens + "]";
}
	
}

File: kindergarden-front-end/JS/components/Parent.js
import Childs from './Childs'


export default function Parent(parent) {
    return `
    <div class="parent__container">
        <h3 class="parent__parentName name">Parent: ${parent.firstName} ${parent.lastName}</h3>
        <h4 class="parent__parentPhoneNumber phone">Phone: ${parent.phoneNumber}</h4>
        <h4 class="parent__parentEmail email">e-mail: ${parent.email}</h4>
    
        <ul class="children">
         
            <li class="children">${Childs(parent.child)}</li>
        </ul>

        

        
    </div>
            `
}       